config.yaml

```diff
@@ -3,6 +3,9 @@ enableRobotsTxt: true
 
 theme: [docsy]
 
+# Title of the project
+title: Notary Project
+
```

Comment:
A few comments:

- Since there's a base setup for multilingual support, the `title` field should be under:
  ```yaml
  languages:
    en:
  ```
  This will avoid problems when another language is introduced.
- Given that the navbrand logo (in the top nav) is "Notary", would it make sense for the site title to be "Notary" as well?
- I'd vote to drop the comment.

Reply:
Sure...I agree with your comments.
Will push a fix right away!